modules/limelightDigitalBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import {ajax} from '../src/ajax.js';
+import * as utils from '../src/utils.js';
+
+const BIDDER_CODE = 'limelight-digital';
```

Comment:
The bidder codes and the names used on the bidder files should be consistent.  If you're in a renaming state, can you please change one or the other so they match?

Reply:
Hi, @jsnellbaker 

I fixed `isBidRequestValid()`. You can use zero adUnitId in tests. Thanks you for report.

I use 'limelight-digital' everywhere instead of 'project-limelight' except one test domain. And I use 'limelightDigital' as a file name prefix everywhere. It is correct?

Thanks you for your work

Reply:
feel free to message here if any changes required

Reply:
No I meant the prefix for your files should match to the bidder code.  Could you change it to `limelightDigital`?

Reply:
Oh, I fixed it. Thank you.